Dockerfile.v4.2.x-armv7l

```diff
@@ -1,18 +1,24 @@
 FROM ubuntu:16.04
 LABEL maintainer="Matt Bentley <mbentley@mbentley.net>"
 
+ARG DEBIAN_FRONTEND=noninteractive
+ARG OMADA_VER=4.2.8
+ARG OMADA_TAR="Omada_SDN_Controller_v${OMADA_VER}_linux_x64.tar.gz"
+ARG OMADA_URL="https://static.tp-link.com/2020/202012/20201211/${OMADA_TAR}"
+
+# download installation tarfile
+ADD ${OMADA_URL} /tmp/${OMADA_TAR}
+
 # install omada controller (instructions taken from install.sh); then create a user & group and set the appropriate file system permissions
 RUN \
   echo "**** Install Dependencies ****" &&\
   apt-get update &&\
-  DEBIAN_FRONTEND="noninteractive" apt-get install --no-install-recommends -y gosu mongodb net-tools openjdk-8-jre-headless tzdata wget &&\
+  apt-get install --no-install-recommends -y gosu mongodb-server-core net-tools openjdk-8-jre-headless tzdata wget &&\
```

Comment:
`mongodb-server-core` isn't a package in the armv7l Ubuntu repos which is why I had to go with `mongodb`.

Reply:
my mistake; fixed